assets/styles/landing.scss

```diff
@@ -28,6 +34,31 @@
     width: 100%;
 }
 
+.features {
+    align-items: center;
+    background: $gallery;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-around;
+
+    .feature {
+        color: $dusty-gray;
+        font-weight: bold;
+        max-width: 250px;
+        text-align: center;
+        width: 100%;
+
+        .img {
+            margin: 40px 35px 0px 35px;
```

Comment:
when the value is 0 you can omit the unit (in this case px), that's because 0 is the same for every length. the last 35px can be removed too since the second parameter sets the left and right margins when there are 3 values.